hugegraph-server/hugegraph-test/src/main/java/org/apache/hugegraph/core/TaskCoreTest.java

```diff
@@ -686,7 +686,7 @@ private HugeTask<Object> runGremlinJob(String gremlin) {
     }
 
     private static void sleepAWhile() {
-        sleepAWhile(100);
+        sleepAWhile(1000);
```

Comment:
Instead of modifying this sleepAWhile() method, can we also call `sleepAWhile(200 * 6)` in testGremlinJobAndCancel(), just like testGremlinJobAndRestore()

Reply:
Thanks for the suggestion, it is indeed more reasonable！